src/components/Authorization/Authorization.jsx

```diff
@@ -32,114 +33,168 @@ export const Authorization = ({primary}) => {
+                  }
+                  onClick={togglePasswordVisibility}
+                />
+              </div>
             </form>
-            <img src={passwordVisible ? eyeIconOpen : eyeIcon} alt="eye icon"
-                 className={'eye-icon' +
-                   (passwordError ? ' password-error-icon' : '') +
-                   (emailError ? ' email-error-icon' : '') +
-                   (emailError && passwordError ? ' both-error-icon' : '')}
-                 onClick={togglePasswordVisibility} />
-              <CommonButton  className='btn' variant="primary" onClick={handleSubmit}>Войти</CommonButton>
-            <div className="links-block" >
+            <CommonButton
+              className="btn-enter"
+              variant={"primary"}
+              onClick={handleSubmit}
+            >
+              Войти
+            </CommonButton>
+            <div className="links-block">
               <ul>
-                <li>Запомнить меня <input id="yes" type="checkbox" name="rememberMe"  checked={rememberMe} onChange={handleRememberMeChange}   /></li>
-                <li className='medium'> Нет аккаунта на IT Roast?</li>
+                <li>
+                  Запомнить меня{" "}
+                  <input
+                    id="yes"
+                    type="checkbox"
+                    name="rememberMe"
+                    checked={rememberMe}
+                    onChange={handleRememberMeChange}
+                  />
+                </li>
+                <li className="medium"> Нет аккаунта на IT Roast?</li>
                 <li>Остались вопросы?</li>
               </ul>
               <ul>
-                <li><Link className="hidden" to={Path.CHANGE_PASSWORD}>Забыли пароль?</Link></li>
-                <li className='medium'><Link to={Path.REGISTRATION} >Присоединяйся!</Link></li>
-                <li><a href="mailto:google.com">Спроси нас!</a></li>
+                <li>
+                  <Link className="hidden-link" to={Path.CHANGE_PASSWORD}>
+                    Забыли пароль?
+                  </Link>
+                </li>
+                <li className="medium">
+                  <Link to={Path.REGISTRATION}>Присоединяйся!</Link>
+                </li>
+                <li>
+                  <a href="mailto:google.com">Спроси нас!</a>
+                </li>
               </ul>
-              <div>
-              </div>
+              <div></div>
             </div>
-            <p>Нажимая кнопку «Войти», вы подтверждаете своё согласие с условиями обработки данных.</p>
+            <p>
+              Нажимая кнопку «Войти», вы подтверждаете своё согласие с условиями
+              обработки данных.
+            </p>
           </div>
         </div>
       </div>
-      <Footer mainPageData={testMainPageData}/>
+      <Footer mainPageData={testMainPageData} />
     </div>
   );
 };
-
 Authorization.propTypes = {
   primary: PropTypes.bool,
 };
```

Comment:
Why we use this? We should make this file .tsx and try to use typescript.

Reply:
in the next component i will try to use ts